plugins/imgur_a.js

```diff
@@ -76,7 +84,7 @@ hoverZoomPlugins.push({
                                                 caption += ';\n';
                                             }
                                             caption += img.description;
```

Comment:
I think nulls are coming from this line. If it's possible to filter them here (just compare img.description with "null"), then you don't need to do replace later.